packages/i18n/I18nDecorator/getI18nClasses.js

```diff
@@ -1,14 +1,21 @@
 import {isNonLatinLocale, isRtlLocale} from '../locale';
 import LocaleInfo from '../ilib/lib/LocaleInfo';
 
+/**
```

Comment:
This doesn't need a jsdoc block, it's not just private, it's not exported from the module at all.